packages/Atoms/Icon/icon-library.js

```diff
@@ -0,0 +1,44 @@
+const iconProps = {}
```

Comment:
In dev mode, hot reload will run code multiple times if you edited the config file, so this global var value may will be kept instead of reset to `{}`.

Reply:
Added ability to invoke a reset to these properties and added it before the first custom library is called. This allows require.context() calls be reset in dev hot reloading.